### DISCUSSION TITLE
[RFC Call] Agenda - January 12, 2022

### DISCUSSION THREAD

**Discussion by FredKSchott** [withastro/roadmap#64] [Announcements]
# Agenda

* @jonathantneal - attempt consensus - https://github.com/withastro/rfcs/pull/63
* @FredKSchott - collect feedback - https://github.com/withastro/rfcs/discussions/65
* @FredKSchott - collect feedback - https://github.com/withastro/rfcs/discussions/67

--- 

_If there is something that you would like to see discussed, please comment below. We will prioritize pending RFC, but if we have time to spare we will open the call up to get attention on and discuss [other threads and feature requests.](https://github.com/withastro/rfcs/discussions)_
If there is something else you would like to discuss, please mention it here.

**Comment by aFuzzyBear** [comment]
Could we also include #69 alongside the discussions pertaining to rfc #67, I feel that the former is a suitable rfc proposal given the feedback and concerns that have been raised so far in the latter part of the  rfc #67 discussions.

**Comment by jonathantneal** [comment]
* https://github.com/withastro/rfcs/pull/63

This proposal was accepted. The general feedback was that it improved consistency. There was some feedback that the proposal was good on merits, but needed more examples to strengthen its case.

* https://github.com/withastro/rfcs/discussions/65

Feedback was mixed, generally positive. There were questions about what the current "magic" is for `hoist`, which may be good to list in a formal proposal. There were questions about where automatically 'hoisted' scripts would be hoisted. One core member voiced a desire to have `<script>` and `<style>` tags require an opt-in attribute, rather than the other way around.

* https://github.com/withastro/rfcs/discussions/67

Feedback was mixed. There was some confusion over whether the proposal should focus on the requirement of a `client` directive or the default for a `client` directive. A couple active contributors voiced a desire to not change the default behavior. One contributor opted for the behavior to exist in import assertions, and the response was that import assertions were not incompatible or alternative to this discussion if they could define a default behavior.
